living_documentation_generator/github_projects.py

```diff
@@ -6,50 +28,53 @@
 
 from living_documentation_generator.model.github_project import GithubProject
 from living_documentation_generator.model.project_issue import ProjectIssue
-from living_documentation_generator.utils.github_project_queries import GithubProjectQueries
+from living_documentation_generator.utils.github_project_queries import (
+    GithubProjectQueries,
```

Comment:
Is needed to have this format when only one import is present?

Reply:
That was a mistake, nice catch. Solved in: